src/libraries/System.Diagnostics.Process/tests/ProcessStartInfoTests.cs

```diff
@@ -472,8 +472,11 @@ public void TestUserCredentialsPropertiesOnWindows()
             {
                 p = CreateProcessLong();
 
-                // ensure the new user can access the .exe (otherwise you get Access is denied exception)
-                SetAccessControl(username, p.StartInfo.FileName, add: true);
+                if (PlatformDetection.IsNotWindowsServerCore) // for this particular Windows version it fails with Attempted to perform an unauthorized operation (#46619)
```

Comment:
The test is still valid without this?  If the test succeeds without this on Windows Server Core, why is it needed on other Windows?

Reply:
I've added this recently when I was working on re-enabling this test and it was mandatory on my machine to make it work (#46083)

I have no idea why it does not work on machines from this certain pool. Is it a thing related to Windows Server? Some custom helix machine setup?

The test is restricted to admin accounts only:

https://github.com/dotnet/runtime/blob/f6e73ad8c8a3252a5528724fa241fca410295008/src/libraries/System.Diagnostics.Process/tests/ProcessStartInfoTests.cs#L454

https://github.com/dotnet/runtime/blob/f6e73ad8c8a3252a5528724fa241fca410295008/src/libraries/System.Diagnostics.Process/tests/ProcessStartInfoTests.cs#L28-L29

Which makes it even more confusing (I would assume that Admin accounts which can create other user accounts can also add permissions to files?)